pkg/ddb/repository.go

```diff
@@ -78,12 +78,21 @@ func NewRepository(config cfg.Config, logger mon.Logger, settings *Settings) Rep
 	settings.AutoCreate = config.GetBool("aws_dynamoDb_autoCreate")
 	settings.Client.MaxRetries = config.GetInt("aws_sdk_retries")
 
+	tableName := namingStrategy(settings.ModelId)
+
 	tracer := tracing.ProviderTracer(config, logger)
 	client := ProvideClient(config, logger, settings)
 
+	backoffSettings := &cloud.BackoffSettings{}
+	config.UnmarshalKey("ddb.backoff", backoffSettings)
+
+	if err := cfg.Merge(backoffSettings, settings.Backoff); err != nil {
+		logger.Fatalf(err, "could not merge backoff settings for ddb table %s", tableName)
```

Comment:
Why do we need to kill the whole server in this case? Shouldn't `logger.Panicf` be enough?

Reply:
true